logic.js

```diff
@@ -0,0 +1,117 @@
+// const postIdCounter = function(posts){
+//     let postsCounter = 0
+//     for (let post in posts)
+//     postsCounter++
+//     return postsCounter
+// }
+
+// let commentIdCounter = function(posts){
+//     let commentCounter = 0
+//     for (let comment in posts.comments)
+
+// }
+
+
+const Tweeter = function () {
+    let _posts = [
```

Comment:
You can use `const` because you are not changing this variable. Use const as much as you can.